README.md

```diff
@@ -148,6 +148,9 @@ positional arguments:
     parse               Debug a results file by parsing and printing each record in it
     summary             Generate a terminal-based summary report of the functions that allocate most memory
     stats               Generate high level stats of the memory usage in the terminal
+    transform           Generate reports files in different formats
+    attach              Begin tracking allocations in an already-started process
+    detach              End the tracking started by a previous ``memray attach`` call
```

Comment:
Would you mind splitting this change out into a separate commit of its own, since it's for a separate purpose than the rest of this PR?